data/mods/gen5/moves.js

```diff
@@ -93,6 +93,21 @@ let BattleMovedex = {
 	},
 	bounce: {
 		inherit: true,
+		effect: {
+			onTryImmunity(target, source, move) {
+				if (move.id === 'gust' || move.id === 'twister') {
```

Comment:
Maybe even assigning ``['gust', 'twister', 'skyuppercut', 'thunder', 'hurricane', 'smackdown',  'thousandarrows', 'helpinghand']`` into a constant variable would be even more readable. 
```suggestion
				if (['gust', 'twister', 'skyuppercut', 'thunder', 'hurricane', 'smackdown',  'thousandarrows', 'helpinghand'].includes(move.id)) {
```